app/views/shared/components/project_list/_project_list.html.erb

```diff
@@ -14,7 +14,7 @@
                 <div class="dropdown-fancy-content">
                   <ul>
                     <li>
-                      <button class="button-naked project-list-item-toggle" type="button" name="button" data-bind="click: toggleVisibility">
+                      <button class="button-naked project-list-item-toggle" type="button" name="button" data-bind="click: function() { toggleVisibility() }">
```

Comment:
What is the reason for the anonymous function here?

Reply:
When using the function directly in the click, it was trying to pass the scope of the button as a parameter in the `toggleVisibility` function call, which was causing some issues with the way the function was called. This seemed like a good way to prevent that from happening.

Reply:
👍